plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/AvroSchemaConverter.java

```diff
@@ -58,6 +59,8 @@
 
 public class AvroSchemaConverter
 {
+    private static final SchemaFormatter JSON_PRETTY_FORMATTER = SchemaFormatter.getInstance("json/pretty");
```

Comment:
Why not use `new JsonSchemaFormatter(true)`?

Reply:
The factory is mentioned jn the docs so I wanted to use most "canonical" way

Reply:
<img width="773" alt="Screenshot 2024-08-06 at 09 21 58" src="https://github.com/user-attachments/assets/274e8a34-ce80-4fa5-9958-c20d24eee7fc">